### PR TITLE
Add font style

### PR DESCRIPTION
**Pull request by ZvonimirMedak**
## Summary

I have added the font style which uses the system font.

### Type

- [X] **Feature**: This pull request introduces a new feature.
- [ ] **Bug fix**: This pull request fixes a bug.
- [X] **Refactor**: This pull request refactors existing code.
- [ ] **Documentation**: This pull request updates documentation.
- [ ] **Other**: This pull request makes other changes.

#### Additional information

- [ ] This pull request introduces a **breaking change**.

### Description

I added a new font style which can be updated as we go, and if we need other font styles. It uses the system font in the background.
I also updated titles on the Device, and Tools screen since they were duplicated.

## Checklist

- [X] I have performed a self-review of my own code.
- [X] I have tested my changes, including edge cases.
- [X] I have added necessary tests for the changes introduced (if applicable).
- [X] I have updated the documentation to reflect my changes (if applicable).

## Additional notes

### CODE REVIEW

File: Sentinel/Classes/Core/Internal/Font+Size.swift [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+import Foundation
+import SwiftUI
+
+enum FontSize: CGFloat {
+    case title1 = 24
+    case subtitle1 = 16
+    case body1 = 14
+    case body2 = 13
+    case caption1 = 11
```
Comment (nikolamajcen) [inline Sentinel/Classes/Core/Internal/Font+Size.swift:16]: Honestly, I would simplify that a bit; remove numbers and put only one `body`. It will make things easier to maintain and use.

**Review by nikolamajcen — APPROVED** [review]
Approving this one, but check the comment and revisit the code if needed.
